scripts/deploy/execute-deployment-pipeline.mjs

```diff
@@ -16,6 +17,23 @@ import shopifyJson from '../../packages/shopify/package.json' with {
   type: 'json',
 };
 
+const packagesAndVersions = [
+  {packageName: 'BUENO', version: buenoJson.version, s3Dir: 'bueno'},
+  {packageName: 'HEADLESS', version: headlessJson.version, s3Dir: 'headless'},
+  {packageName: 'ATOMIC', version: atomicJson.version, s3Dir: 'atomic'},
+  {
+    packageName: 'ATOMIC_REACT',
+    version: atomicReactJson.version,
+    s3Dir: 'atomic-react',
+  },
+  {
+    packageName: 'ATOMIC_HOSTED_PAGE',
+    version: atomicHostedPageJson.version,
+    s3Dir: 'atomic-hosted-page',
+  },
+  {packageName: 'SHOPIFY', version: shopifyJson.version, s3Dir: 'shopify'},
+];
+
 function getVersionComposants(version) {
```

Comment:
A bit out of scope, but I'd rename this on function to `getVersionComponents` ("composants" isn't an English word).